src/dbnode/storage/bootstrap/bootstrapper/commitlog/source_prop_test.go

```diff
@@ -594,17 +602,28 @@ func genWrite(start time.Time, bufferPast, bufferFuture time.Duration, ns string
 			a = a.Add(bufferPast)
 		}
 
+		tags := seriesUniqueTags(id, tagKey, tagVal, includeTags)
+		tagSliceIter.Reset(tags)
+
+		tagEncoder := tagEncoderPool.Get()
+		err := tagEncoder.Encode(tagSliceIter)
+		require.NoError(t, err)
+
+		encodedTagsChecked, ok := tagEncoder.Data()
+		require.True(t, ok)
```

Comment:
nit: I think it's better to panic here rather than flowing through `t`, should achieve the same result